src/misc.c

```diff
@@ -30,8 +30,8 @@ void ecs_sleepf(
     double t)
 {
     if (t > 0) {
-        int sec = t;
-        int nsec = ((t - sec) * 1000000000);
+        int sec = (int)t;
```

Comment:
Are implicit conversions not allowed in MSVC?

Reply:
The implicit cast converts double to an int, warns about loss of data:

Warning	C4244	'initializing': conversion from 'double' to 'int', possible loss of data	

imo, this sort of casting should always be explicit